### PR TITLE
Fix contributing shellcheck

### PR DESCRIPTION
**Pull request by Bill-Kunj**

### CODE REVIEW

**Review by dckc — COMMENTED** [review]
I'm not interested to maintain shell scripts; I'd rather get rid of them rather than document them more. I'll leave this for someone else to reivew.

**Comment by Bill-Kunj** [comment]
> I'm not interested to maintain shell scripts; I'd rather get rid of them rather than document them more. I'll leave this for someone else to reivew.

@dckc Getting rid of them is the reason for #217. Help is needed to figure that out. #217 simplifies the re-implementation of the scripts, and the implementation of a non-developer interface for RGOV.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This project uses [```shellcheck```](https://github.com/koalaman/shellcheck) for
 Shell script contributors using vscode are encouraged to install the vscode extension https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck.
 ```shellcheck``` is also available in the Ubuntu package ```shellcheck```
 
+[We](https://docs.google.com/document/d/1UfoBMB_OFm-pkjid0u9d0GddB2bWXjc8u-bo4Es0isU/edit#) also suggest all contributions should pass:
```
Comment (dckc) [inline CONTRIBUTING.md:11]: What is the relevance if rdev?

**Review by dckc — COMMENTED** [review]
In what way is this a fix? What was broken?

**Comment by dckc** [comment]
This seems to be merged without peer review. What motivated that?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Bill-Kunj) [reply CONTRIBUTING.md:11]: Consult with @jimscarver for details abut the RDev reference.

**Comment by Bill-Kunj** [comment]
> In what way is this a fix? What was broken?
@dckc it's just a name for the branch - breaking it down semantically is unnecessary.

**Comment by Bill-Kunj** [comment]
> This seems to be merged without peer review. What motivated that?

https://github.com/rchain-community/rgov/pull/240#pullrequestreview-703643516 resulted in face-to-face review with other RGOV members. I merged it out of convenience and consideration for other people's time.

**Comment by dckc** [comment]
> > This seems to be merged without peer review. What motivated that?
> 
> [#240 (review)](https://github.com/rchain-community/rgov/pull/240#pullrequestreview-703643516) resulted in face-to-face review with other RGOV members. I merged it out of convenience and consideration for other people's time.

OK, so there was peer review. Cool.

In the future, if you're acting on behalf of others, go ahead and make a note to leave an audit trail, please.

File: CONTRIBUTING.md (hunk 1 above)
Comment (dckc) [reply CONTRIBUTING.md:11]: @jimscarver ? What is the relevance of rdev?

**Comment by jimscarver** [comment]
the project is being performed by rdev decentralized governance. rdev is
the testbed for rgov.

**Comment by dckc** [comment]
> the project is being performed by rdev decentralized governance. rdev is the testbed for rgov.

While rdev is as welcome as anyone to use rgov, rgov is not in any way controlled or owned or subsumed by rdev. The project is being performed by its contributors.

I thought I was pretty clear about this in the [2020-12-07 rdev meeting](https://docs.google.com/document/d/1NImbkoxPoL5hI4GtTl6ziOC8TcvdeSUOEwlz4hViV34/edit#heading=h.t3i2dangaxrx), though I evidently didn't manage to get it in the record.

**Pull request merged**